Maths/test/ArmstrongNumber.test.ts

```diff
@@ -0,0 +1,27 @@
+import { ArmstrongNumber } from "../ArmstrongNumber"
+
+describe('ArmstrongNumber', () => {
+	it('should return the correct value', () => {
```

Comment:
See https://github.com/TheAlgorithms/TypeScript/blob/master/CONTRIBUTING.md#writing-good-tests - this seems like a perfect use case for `each`.